### PR TITLE
mis_builder: add changelog

### PR DESCRIPTION
**Pull request by sbidoul**
wait for
- [x] #177 
- [x] #170 
- [x] #169 
- [x] #167

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+.. Future (?)
+.. ~~~~~~~~~~
+.. 
+.. * 
+
+8.0.1.0.0 (2016-03-xx)
+~~~~~~~~~~~~~~~~~~~~~~
+
+* Usability: the default view for MIS Report instances is now the rendered preview,
+  and the settings are accessible through an gear icon in the list view and
```
Comment (laetitia-gangloff) [inline CHANGES.rst]: "a gear icon" instead of "an gear icon"

File: CHANGES.rst [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  a button in the preview.
+  https://github.com/OCA/account-financial-reporting/pull/170
+* Display blank cells instead of 0.0 when there is no data.
+  https://github.com/OCA/account-financial-reporting/pull/169
+* Usability: better layout of the MIS Report periods settings on small screens.
+  https://github.com/OCA/account-financial-reporting/pull/167
+* Add a mis_build_analytic_filter module to demonstrate analytic filtering.
+  https://github.com/OCA/account-financial-reporting/pull/167
+* Include the download buttons inside the MIS Builder widget, and refector
```
Comment (laetitia-gangloff) [inline CHANGES.rst]: refactor instead of refector

**Comment by sbidoul** [comment]
This one is ready to review and merge.

**Comment by sebastienbeau** [comment]
:+1:

**Comment by tafaRU** [comment]
:+1:

**Comment by adrienpeiffer** [comment]
:+1:

**Pull request merged**